rir/src/interpreter/interp.cpp

```diff
@@ -1307,6 +1344,25 @@ SEXP evalRirCode(Code* c, Context* ctx, SEXP* env, const CallContext* callCtxt,
             NEXT();
         }
 
+        INSTRUCTION(mk_stub_env_) {
+            size_t n = readImmediate();
+            advanceImmediate();
+            // Do we need to preserve parent and the arg vals?
+            SEXP parent = ostack_pop(ctx);
+            assert(TYPEOF(parent) == ENVSXP &&
+                   "Non-environment used as environment parent.");
+            advanceImmediateN(n);
+            auto args = new std::vector<SEXP>(n);
+            for (size_t i = 0; i < n; ++i) 
+                args->push_back(ostack_pop(ctx));
```

Comment:
I am not convinced that this is sound. In most cases it will be, because we keep a copy of function arguments when we do a call. But for inlined `mk_envs_` I believe that you will hide the arguments from the GC and this will result in the gc collecting them.

Reply:
No, It is not. We must find a way to trace those objects. My point is that in our current code this problem does not raise. So maybe we can think on a good design and implement it later. Maybe we should change the `RirWrappers` to `externalSPX`. The problem is twofold. On one hand we do not want to introduce an overhead to the gc. On the other we do not want to lose to much time with the wrapper creation, otherwise the optimization starts to lose sense.